src/docs/issue-platform-detectors.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: "Issue Platform - Writing Detectors"
+---
+
```

Comment:
Since this page is in the sidebar, readers might get here without having read the Issue Platform page first. A very brief opening paragraph that says what a detector is and links back to the main Issue Platform page might help just in case? e.g.

```suggestion

Issue detectors identify application issues by examining one or more datasets collected by Sentry, and report detected issue occurances via the <Link to="/issue-platform/">Issue Platform</Link>. Detectors must fingerprint issues accurately and provide actionable information to developers.
```